File: Client/src/app/models/process.ts
import { Category } from './category';
import { Phase } from '.';

export class Process {
    processId: number;
    name: string;
    description: string;
    createdBy: string;
    createdAt: string;
    updatedAt: string;
    updatedBy: string;
    status: string;
    categoryId: number;
    category: Category;
    phase: Phase[];
}

File: Server/DataAccess/Models/Process.cs
using System;
using System.Collections.Generic;

namespace DataAccess.Models
{
    public partial class Process
    {
        public Process()
        {
            Phase = new HashSet<Phase>();
        }

        public Guid ProcessId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string CreatedBy { get; set; }
        public DateTime? CreatedAt { get; set; }
        public string UpdatedBy { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public string Status { get; set; }
        public Guid CategoryId { get; set; }

        public virtual Category Category { get; set; }
        public virtual ICollection<Phase> Phase { get; set; }
    }
}
